performance/vip-tweaks.php

```diff
@@ -80,3 +80,77 @@ function wpcom_vip_media_library_months_with_files() {
 		return $months;
 	}
 }
+
+/**
+ * Improves performance of all the post listing pages that display the months dropdown filter.
+ * A unique transient is created for each post type.
+ */
+if ( is_admin() ) {
+
+	add_filter( 'pre_months_dropdown_query', 'wpcom_vip_pre_months_dropdown_query', 10, 2 );
+	function wpcom_vip_pre_months_dropdown_query( $months, $post_type ) {
+		global $wpdb;
+		
+		// Don't set a transient for trashed post.
+		if ( isset( $_GET['post_status'] ) && "trash" === $_GET['post_status'] ) {
```

Comment:
When calling $_GET['post_status'] several times, we should assign it to a variable and saniitize it